core/src/test/resources/spark-events/local-1422981780767

```diff
@@ -58,7 +58,7 @@
+{"Event":"SparkListenerJobEnd","Job ID":1,"Job Result":{"Result":"JobFailed","Exception":{"Message":"Job aborted due to stage failure: Task 3 in stage 2.0 failed 1 times, most recent failure: Lost task 3.0 in stage 2.0 (TID 19, localhost): java.lang.RuntimeException: got a 3, failing\n\tat $line11.$read$$iwC$$iwC$$iwC$$iwC$$anonfun$1.apply(<console>:18)\n\tat $line11.$read$$iwC$$iwC$$iwC$$iwC$$anonfun$1.apply(<console>:17)\n\tat scala.collection.Iterator$$anon$11.next(Iterator.scala:328)\n\tat org.apache.spark.util.Utils$.getIteratorSize(Utils.scala:1311)\n\tat org.apache.spark.rdd.RDD$$anonfun$count$1.apply(RDD.scala:910)\n\tat org.apache.spark.rdd.RDD$$anonfun$count$1.apply(RDD.scala:910)\n\tat org.apache.spark.SparkContext$$anonfun$runJob$4.apply(SparkContext.scala:1314)\n\tat org.apache.spark.SparkContext$$anonfun$runJob$4.apply(SparkContext.scala:1314)\n\tat org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:61)\n\tat org.apache.spark.scheduler.Task.run(Task.scala:56)\n\tat org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:196)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)\n\tat java.lang.Thread.run(Thread.java:745)\n\nDriver stacktrace:","Stack Trace":[]}}}
```

Comment:
There's absolutely no point in changing this file.

Reply:
The event log info here is not used currently. I'll revert this change.

Would you please explain a bit more on why we should keep the old test data? I'd like to learn the philosophy behind your comments in case I meet with similar scenarios in the future. Is this an example of "If It Ain't Broke, Don't Fix It"? Or do we keep the old test data here for backward compatibility?

Reply:
Let me throw the question back at you: why do you think it's important to change this?

And why only remove references to akka from these files, and not to every single thing that has changed since they were created?

(The answer to the last question will hopefully show you why there's no point in changing this.)

Reply:
I agree this is much more about HistoryServer, not about Akka . Thank you for you explanation.

BTW, does it make sense to regenerate these event logs using the latest Spark (of course with another PR)?

Reply:
Ask yourself the question: what would doing that achieve?

Reply:
It seems I need to do some homework to answer this question. Many thanks for your tough and helpful questions!